source/globalCommands.py

```diff
@@ -557,19 +558,26 @@ def script_toggleReportFontSize(self,gesture):
 
 	@script(
 		# Translators: Input help mode message for toggle report font attributes command.
-		description=_("Toggles on and off the reporting of font attributes"),
+		description=_("Toggles between speaking, brailling, speaking and brailling, and not reporting font attributes."),
```

Comment:
Alternative wording to put the main subject at the start

```suggestion
		description=_("Cycles font attributes reporting between speech, braille, speech and braille, and off")
```